grafana-performance-overview-dashboard.md

```diff
@@ -93,39 +99,46 @@ Queries Using Plan Cache OPS：所有 TiDB 实例每秒使用 Plan Cache 的查
 - 99： 所有请求命令的 P99 执行时间
 - avg by type：按 `SELECT`、`INSERT`、`UPDATE` 类型统计所有 TiDB 实例上所有请求命令的平均执行时间
 
-## Connection Idle Duration
+### Connection Idle Duration
 
 Connection Idle Duration 指空闲连接的持续时间。
 
 - avg-in-txn：处于事务中，空闲连接的平均持续时间
 - avg-not-in-txn：没有处于事务中，空闲连接的平均持续时间
 - 99-in-txn：处于事务中，空闲连接的 P99 持续时间
+
+### Connection Count
+
+- total：所有 TiDB 节点的总连接数
+- active connections：所有 TiDB 节点的总活跃连接数
+- tidb-<node-number>-peer：各个 TiDB 节点的连接数
```

Comment:
```suggestion
- tidb-\<node-number\>-peer：各个 TiDB 节点的连接数
```
please escape `<>`; otherwise ⬇️

<img width="424" alt="image" src="https://user-images.githubusercontent.com/59654584/208575598-eeb43fbc-354d-46b6-9029-6a1a66751ba3.png">